### PR TITLE
ParseEmailFiles: Do not decode attachments

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40738

## Description
We tried to decode attachments to utf-8, which is wrong.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jochman** [comment]
@ChanochShayner  is there a reason to decode the attachments to utf-8?

**Comment by ChanochShayner** [comment]
> @ChanochShayner is there a reason to decode the attachments to utf-8?

I just changed it:
image

**Comment by Itay4** [comment]
@jochman can we add unit test that fails to decode a zip ?

File: Packs/CommonScripts/ReleaseNotes/1_4_28.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### UnzipFile
```
Comment (Itay4) [inline Packs/CommonScripts/ReleaseNotes/1_4_28.md:4]: fix script name

File: Packs/CommonScripts/ReleaseNotes/1_4_28.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
+<<<<<<< HEAD
```
Comment (Itay4) [inline Packs/CommonScripts/ReleaseNotes/1_4_28.md:1]: fix

File: Packs/CommonScripts/pack_metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.4.28",
+    "currentVersion": "1.5.0",
```
Comment (Itay4) [inline Packs/CommonScripts/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.4.29",
```

File: Packs/CommonScripts/ReleaseNotes/1_4_29.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
 #### Scripts
 ##### GetIndicatorDBotScoreFromCache
 - Fixed an issue where the automation script failed to run on indicators with a verdict of *Unknown*.
-- Updated the Docker image to: *demisto/python3:3.9.6.24019*.
\ No newline at end of file
+- Updated the Docker image to: *demisto/python3:3.9.6.24019*.
```
Comment (Itay4) [inline Packs/CommonScripts/ReleaseNotes/1_4_29.md:4]: revert this

File: Packs/CommonScripts/ReleaseNotes/1_4_30.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#### Scripts
+
+##### ParseEmailFiles
+
+- Fixed an issue where non utf-8 attachments were decoded to utf-8.
```
Comment (bakatzir) [inline Packs/CommonScripts/ReleaseNotes/1_4_30.md:5]:
```suggestion
##### ParseEmailFiles
- Fixed an issue where non UTF-8 attachments were decoded to UTF-8.
```

**Pull request merged**